### PR TITLE
Changed ConsentiumInc to ConsentiumIoT

### PR DESCRIPTION
**Pull request by ConsentiumIoT**
The Github user name was changed from ConsentiumInc to ConsentiumIoT.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @ConsentiumIoT. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/09/19 15:48:27 Loaded 7314 libraries from DB
Changing URL of library ConsentiumThings from https://github.com/ConsentiumInc/ConsentiumThings.git to https://github.com/ConsentiumIoT/ConsentiumThings.git
Success!
```
```
2024/09/19 15:47:06 Loaded 7314 libraries from DB
Changing URL of library EdgeNeuron from https://github.com/ConsentiumInc/EdgeNeuron.git to https://github.com/ConsentiumIoT/EdgeNeuron.git
Success!
```

**Pull request merged**
